.github/workflows/veedub.yaml

```diff
@@ -8,7 +8,7 @@ on:
         type: string
       DOCKER_CONTAINER_REGISTRY_USER:
         required: false
-        default: lukepon
```

Comment:
`lukepon` user no longer exists, org-level key for ponbike has been updated to match `pondsdevops`.